rest-assured-tests/src/test/java/TariffManagementServiceTests.java

```diff
@@ -0,0 +1,28 @@
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static io.restassured.RestAssured.*;
+import static org.junit.Assert.assertEquals;
+
+public class TariffManagementServiceTests {
+    private static int tariffsNumber;
+
+    @BeforeClass
+    public static void configureRestAssured() {
+        port = 8092;
+        tariffsNumber = TariffManagementServiceClass.getTariffsList().length;
+    }
+
+    @Test
+    public void testNewTariffSuccessfulCreation() {
+        Tariff newTestTariff = TariffManagementServiceClass.createTariffByPost("NewTestTariff", 4.67f, 12);
+
+        assertEquals(tariffsNumber + 1, TariffManagementServiceClass.getTariffsList().length);
```

Comment:
write javadoc or create a separate variable for "tariffs + 1" so the purpose of this evaluation can be understandable

Reply:
javadoc added